cnxepub/html_parsers.py

```diff
@@ -16,11 +18,13 @@
     }
 
 
-def _squash_to_text(elm):
+def _squash_to_text(elm, remove_namespaces=False):
     value = [elm.text or '']
     for child in elm.getchildren():
-        value.append(etree.tostring(child).decode('utf-8'))
+        value.append(etree.tostring(child).decode('utf-8').strip())
         value.append(child.tail or '')
+    if remove_namespaces:
+        value = [re.sub(' xmlns:?[^=]*="[^"]*"', '', v) for v in value]
```

Comment:
I don't see a way to remove namespaces.  If you don't like it I guess we don't have to remove the namespaces.

Why do you think this only allows one level of markup?